presto-tests/src/test/java/com/facebook/presto/execution/TestEventListener.java

```diff
@@ -165,6 +166,32 @@ public void testNormalQuery()
         assertEquals(queryCompletedEvent.getStatistics().getTotalRows(), expectedCompletedPositions);
     }
 
+    @Test
+    public void testOutputStats()
+            throws Exception
+    {
+        int expectedEvents = 1 + 1 + SPLITS_PER_NODE + 1 + 1;
```

Comment:
Can you add the same comment that the previous method has? The math is really confusing otherwise.